File: Biweekly Contest 113/Q1.java
https://leetcode.com/contest/biweekly-contest-113/problems/minimum-right-shifts-to-sort-the-array/
8039. Minimum Right Shifts to Sort the Array


class Solution {
    public int minimumRightShifts(List<Integer> nums) {
        int count=0;
        int i=0;
        for(i=0;i<nums.size();i++){
            if(nums.get(i)>nums.get((i+1)%nums.size())){
                break;
            }
        }
        
        for(i=i+1;i<nums.size();i++){
            if(nums.get(i)>nums.get((i+1)%nums.size())){
                return -1;
            }
            count++;
        }
        
        return count;
        
    }
}

File: Biweekly Contest 122/Q1.cpp
3010. Divide an Array Into Subarrays With Minimum Cost I
https://leetcode.com/contest/biweekly-contest-122/problems/divide-an-array-into-subarrays-with-minimum-cost-i/

class Solution {
public:
    int minimumCost(vector<int>& nums) {
        int minn1=nums[0];
        sort(nums.begin()+1,nums.end());
        return minn1+nums[1]+nums[2];
    }
};